test/gas-analytics/SpokePool.RelayerRefundRootExecution.ts

```diff
@@ -244,5 +245,32 @@ describe("Gas Analytics: SpokePool Relayer Refund Root Execution", function () {
       const receipt = await txn.wait();
       console.log(`executeRelayerRefundRoot-gasUsed: ${receipt.gasUsed}`);
     });
+    it(`Stress Test: 1 leaf contains ${STRESS_TEST_REFUND_COUNT} refunds with amount > 0`, async function () {
+      // This test should inform the limit # refunds that we would allow a RelayerRefundLeaf to contain to avoid
+      // publishing a leaf that is unexecutable due to the block gas limit.
+
+      // Regarding the block limit, we should target a maximum of 30 million gas. Technically block's can support up
```

Comment:
I think these comments are a little confusing. It makes it sound like we're targeting 30 million. Maybe we should just say that the max limit is 30 million, the expected block gas limit is 15 million, and we use 12 million as a conservative upper-bound.